xdcs-server/src/main/java/pl/edu/agh/xdcs/services/TaskService.java

```diff
@@ -162,13 +174,37 @@ public TaskCreationWizard addResourcePattern(
             return this;
         }
 
+        public TaskCreationWizard addKernelArguments(List<String> kernelArguments) {
```

Comment:
wartości argumentów nie mogą latać jako stringi, to są dane binarne

Reply:
z danymi binarnymi jest problem połączenia Typescript oraz Javy. Jak wrzucę type: string, format: binary to w Javie mam File, a w Typescript Blob

Reply:
i to jest okej z punktu TS, z punkty Javy nigdy mi się nie podobało to, że OpenAPI generuje binary jako `File` a nie na przykład `byte[]` albo `InputStream`, w najgorszym wypadku możesz ustawić na zwykły string i kodować w base64 ręcznie

Reply:
przerobione na type: string, format:bytes: Java -> byte[], Typescript -> string (base64)